xbmc/cores/AudioEngine/Sinks/pipewire/PipewireNode.h

```diff
@@ -65,10 +65,10 @@ class CPipewireNode : public CPipewireProxy
 
   std::unique_ptr<pw_node_info, PipewireNodeInfoDeleter> m_info;
 
-  std::set<spa_audio_format> m_formats;
-  std::set<spa_audio_channel> m_channels;
-  std::set<uint32_t> m_rates;
-  std::set<spa_audio_iec958_codec> m_iec958Codecs;
+  mutable std::set<spa_audio_format> m_formats;
+  mutable std::set<spa_audio_channel> m_channels;
+  mutable std::set<uint32_t> m_rates;
+  mutable std::set<spa_audio_iec958_codec> m_iec958Codecs;
```

Comment:
Design flaw alarm. This is a lot of state to mark mutable. Usually you only want to use mutable to get around "unrelated" state like mutex internals. The constness of the above methods should agree with these being regular variables.